content/design/pages/landing-page.md

```diff
@@ -0,0 +1,215 @@
+>}}"""
+
+form ="""
+{{<featured/promo-tips-form
+  hed="Send A Tip"
+  dek="Help support the Penn State Transparency Tracker by submitting tips or questions to reporter Wyatt Massey — via the submission form, email at [wmassey@spotlightpa.org](mailto:wmassey@spotlightpa.org), or one of the methods described below."
+>}}
+"""
+newsletter = """
+{{<newsletter/pennstatealert-full-form>}}
+"""
+[[blocks]]
+title = "Text Block Two"
+layout = "text-2"
+copy="""
+  {{<featured/large>}}Spotlight PA is a nonpartisan, independent newsroom paid for by grants from foundations and donations from individuals committed to robust investigative reporting. Spotlight PA editors and reporters operate independently of our funders and maintain editorial control over all of the content they produce. Funders do not have input into the selection of which stories Spotlight PA pursues or the reporting process for those stories. Funders do not review any content before publication or have any special access to reporters or newsroom leadership. In short, we answer to you, the public.{{</featured/large>}}
+
+  {{<featured/large>}}Funds for Spotlight PA are donated to the nonprofit <a href="https://www.lenfestinstitute.org/" class="text-yellow">Lenfest Institute for Journalism</a>, which then directs them to our work. As part of our gift acceptance policy, and consistent with our promise to be fully transparent, we do not accept anonymous donations.{{</featured/large>}}
+
+"""
+[blocks.cta]
+url = "/design/"
+copy = "Action in block two"
+
+[[blocks]]
+title = "Promo Block"
+layout = "promo"
+
+[[blocks.entries]]
+image = "2022/12/01hs-hn2r-zez7-rm35.png"
+hed = """
+Option One
+"""
+body = "Interested in becoming a champion of Spotlight PA at a higher level?"
+actionUrl = "https://checkout.fundjournalism.org/memberform?org_id=spotlightpa&theme=leaders"
+actionText = "Become a leader →"
+url = "/support/leaders-in-action"
+
+[[blocks.entries]]
+image = "2022/06/01hb-4958-4rxz-4t0q.png"
+hed = "Option Two"
+body = "Want to support our investigative journalism about Penn State and our State College regional bureau?"
+actionUrl = "https://checkout.fundjournalism.org/memberform?org_id=spotlightpa&theme=centrecounty&campaign=7015G0000013pZTQAY"
+actionText = "Support the Bureau →"
+url = "/support/state-college"
+
+[[blocks]]
+layout = "text-3"
+title = "Text Block Three"
+show_partners = 4
+copy = """
+
+  <p class="md:leading-relaxed">Spotlight PA’s unique model of public-service journalism relies on the generous support of individuals and institutions who understand the vital importance of Spotlight PA’s work to the future of Pennsylvania. The newsroom is supported by a dedicated group of local and national foundations, media partners, individual major donors (“Leaders in Action”), and thousands of readers from across Pennsylvania (“members”). Consistent with our mission, we disclose the source of every dollar we receive, and we do not accept anonymous gifts.</p>
```

Comment:
Similar to above, if we want relaxed leading, it should be part of the text-3 template, not a code on the copy.